easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-90.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'NASM'
+version = '2.13.01'
+
+homepage = 'http://www.nasm.us/'
+
+description = """NASM: General-purpose x86 assembler"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
```

Comment:
The PIC option is not present in any previous easyconfig file. Is there a specific reason why is it now needed?